libs/scripts/delayed.js

```diff
@@ -51,7 +51,10 @@ export const loadPrivacy = async (getConfig, loadScript) => {
 
 export const loadGoogleLogin = async (getMetadata, loadIms, loadScript) => {
   const googleLogin = getMetadata('google-login')?.toLowerCase();
-  if (googleLogin !== 'on' || window.adobeIMS?.isSignedInUser()) return;
+  if (!['mobile', 'desktop', 'on'].includes(googleLogin) || window.adobeIMS?.isSignedInUser()) return;
```

Comment:
King of nits: I would add the logged in check first. If the user is logged in, no sense to check whether googleLogin is enabled.